cluster/cluster.go

```diff
@@ -11,5 +11,7 @@ type Cluster interface {
 	Containers() []*Container
 	Container(IdOrName string) *Container
 
+	Pull(name string, begin, end func(string))
```

Comment:
I didn't get it.

Btw I don't like those begin/end, if you have a better idea I'd love to hear it.
(I think giving the `io.Writer` to pull is a but too much, don't you think ?)